modules/ztp-image-based-upgrade-procedure-cancel.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-procedure-cancel_{context}"]
+= Canceling an image-based upgrade on managed clusters at scale
+
+Cancel the upgrade on a set of managed clusters that completed the `Prep` stage.
```

Comment:
+1

Reply:
Also of note is that the rendering is off because of the plus symbol syntax. Please check the syntax. 

![Screenshot from 2024-09-20 16-15-56](https://github.com/user-attachments/assets/2d4e38f5-fca1-4c28-9afc-ea14bd20ca04)